### PR TITLE
SQL Database Data Masking Policies 

### PR DESCRIPTION
**Pull request by mikhailshilkov**
Resolve https://github.com/pulumi/pulumi-azure-native/issues/605

### CODE REVIEW

File: sdk/go/azure/sql/latest/dataMaskingPolicy.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+			Type: pulumi.String("azure-native:sql:DataMaskingPolicy"),
+		},
+		{
+			Type: pulumi.String("azure-nextgen:sql:DataMaskingPolicy"),
+		},
+		{
+			Type: pulumi.String("azure-native:sql/v20140401:DataMaskingPolicy"),
+		},
+		{
+			Type: pulumi.String("azure-nextgen:sql/v20140401:DataMaskingPolicy"),
```
Comment (lblackstone) [inline sdk/go/azure/sql/latest/dataMaskingPolicy.go:69]: Technically, we don't need to alias new resource types to `azure-nextgen`, but I'm not sure it's worth the effort to special case.

**Review by lblackstone — APPROVED** [review]

File: sdk/go/azure/sql/latest/dataMaskingPolicy.go (hunk 1 above)
Comment (viveklak) [reply sdk/go/azure/sql/latest/dataMaskingPolicy.go:69]: Good catch. Wonder when we can safely cull references to azure-nextgen? Would the 1.0 release be a good time for this?

Comment (lblackstone) [reply sdk/go/azure/sql/latest/dataMaskingPolicy.go:69]: > Would the 1.0 release be a good time for this?

That seems reasonable to me. It's a relatively minor breaking change, and any affected users could simply add the alias themselves.

Comment (mikhailshilkov) [reply sdk/go/azure/sql/latest/dataMaskingPolicy.go:69]: My plan was to cut all of them at 1.0. And yes, I didn't bother to exclude nextgen here.

**Pull request merged**